mautic/mautic
# Region typo smallfix (Cordóba > Córdoba)

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Fixes a typo for Córdoba spanish region.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a record where region is used (lead, company, page hit, video hit, ...).
2. Check "Region"/"State" select values
3. You'll find "Cordóba" which is not a valid name for a spanish region. It should be "Córdoba".

#### Steps to test this PR:
1. Apply PR
2. Clear cache (just in case)
3. Run migrations
4. Check that the new value is "Córdoba" and not "Cordóba" everywhere.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
